cmd/internal/cli/startvm.go

```diff
@@ -72,10 +72,10 @@ func startVM(sifImage, singAction, cliExtra string, isInternal bool) error {
 		sylog.Fatalf("Failed to determine image absolute path for %s: %s", sifImage, err)
 	}
 	if _, err := os.Stat(bzImage); os.IsNotExist(err) {
-		sylog.Fatalf("This functionality is not supported. For more information visit: https://sylabs.io/apptainer-desktop-macos")
+		sylog.Fatalf("This functionality is not supported. For more information visit: https://sylabs.io/singularity-desktop-macos")
```

Comment:
Lets create a follow up issue to remove the singularity desktop (macos vm based containers) from the codebase as it is not maintained anymore

Reply:
done

Reply:
deferred to #52